charts/observability-pipelines-worker/templates/secret-file-backend.yaml

```
@@ -0,0 +1,12 @@
+{{- $secretFile := .Values.bootstrapBackend.secretFile | default dict }}
+
+{{- if $secretFile.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "opw.fullname" . }}-secret-file-backend
```

```suggestion
metadata:
  name: {{ include "opw.fullname" . }}-secret-file-backend
  namespace: {{ .Release.Namespace }}
  labels: {{ include "opw.labels" . | nindent 4 }}
```
I guess we could add namespace and labels to keep all resources' metadata in sync.